### ISSUE TITLE
播放acfun视频 报错

### ISSUE THREAD

**Issue by imzf** [soimort/you-get#479]
you-get -p mpv "http://www.acfun.tv/v/ac1105608"
Video Site: Sina.com
Title:      【晚香玉XIshmael】12 Fanclub【旗袍女仆】 - Part 1
Type:       Flash video (video/x-flv)
Size:       0.0 MiB (0 Bytes)

Traceback (most recent call last):
  File "/usr/local/bin/you-get", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.4/dist-packages/you_get/__main__.py", line 91, in main
    main()
  File "/usr/local/lib/python3.4/dist-packages/you_get/common.py", line 998, in main
    script_main('you-get', any_download, any_download_playlist)
  File "/usr/local/lib/python3.4/dist-packages/you_get/common.py", line 892, in script_main
    download_main(download, download_playlist, args, playlist, output_dir=output_dir, merge=merge, info_only=info_only)
  File "/usr/local/lib/python3.4/dist-packages/you_get/common.py", line 784, in download_main
    download(url, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/you_get/common.py", line 991, in any_download
    m.download(url, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/you_get/extractors/acfun.py", line 70, in acfun_download
    acfun_download_by_vid(p_vid, p_title, output_dir=output_dir, merge=merge, info_only=info_only)
  File "/usr/local/lib/python3.4/dist-packages/you_get/extractors/acfun.py", line 30, in acfun_download_by_vid
    sina_download_by_vid(sourceId, title, output_dir=output_dir, merge=merge, info_only=info_only)
  File "/usr/local/lib/python3.4/dist-packages/you_get/extractors/sina.py", line 33, in sina_download_by_vid
    sina_download_by_xml(xml, title, output_dir, merge, info_only)
  File "/usr/local/lib/python3.4/dist-packages/you_get/extractors/sina.py", line 47, in sina_download_by_xml
    download_urls(urls, title, 'flv', size, output_dir = output_dir, merge = merge)
  File "/usr/local/lib/python3.4/dist-packages/you_get/common.py", line 478, in download_urls
    assert urls
AssertionError

**Comment by soimort** [comment]
@jackyzy823 I failed this one, always after `C10` and `C20`:

```
➜  you-get-develop git:(jackyzy823-fix-acfun) ./you-get -i http://www.acfun.tv/v/ac1105608
Current Video Supports:
	--foramt C10 <URL>: 分段_高清_FLV
	--foramt C20 <URL>: 分段_高清_FLV
Traceback (most recent call last):
  File "/usr/lib/python3.4/urllib/request.py", line 1174, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/usr/lib/python3.4/http/client.py", line 1090, in request
    self._send_request(method, url, body, headers)
  File "/usr/lib/python3.4/http/client.py", line 1128, in _send_request
    self.endheaders(body)
  File "/usr/lib/python3.4/http/client.py", line 1086, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python3.4/http/client.py", line 924, in _send_output
    self.send(msg)
  File "/usr/lib/python3.4/http/client.py", line 859, in send
    self.connect()
  File "/usr/lib/python3.4/http/client.py", line 836, in connect
    self.timeout, self.source_address)
  File "/usr/lib/python3.4/socket.py", line 491, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/usr/lib/python3.4/socket.py", line 530, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./you-get", line 19, in <module>
    you_get.main(repo_path=_filepath)
  File "/home/soimort/Projects/you-get-develop/src/you_get/__main__.py", line 91, in main
    main()
  File "/home/soimort/Projects/you-get-develop/src/you_get/common.py", line 999, in main
    script_main('you-get', any_download, any_download_playlist)
  File "/home/soimort/Projects/you-get-develop/src/you_get/common.py", line 893, in script_main
    download_main(download, download_playlist, args, playlist, output_dir=output_dir, merge=merge, info_only=info_only)
  File "/home/soimort/Projects/you-get-develop/src/you_get/common.py", line 785, in download_main
    download(url, **kwargs)
  File "/home/soimort/Projects/you-get-develop/src/you_get/common.py", line 992, in any_download
    m.download(url, **kwargs)
  File "/home/soimort/Projects/you-get-develop/src/you_get/extractors/acfun.py", line 126, in acfun_download
    acfun_download_by_vid(p_vid, p_title, output_dir=output_dir, merge=merge, info_only=info_only ,**kwargs)
  File "/home/soimort/Projects/you-get-develop/src/you_get/extractors/acfun.py", line 95, in acfun_download_by_vid
    _, _, tmp =url_info(i)
  File "/home/soimort/Projects/you-get-develop/src/you_get/common.py", line 216, in url_info
    response = request.urlopen(request.Request(url))
  File "/usr/lib/python3.4/urllib/request.py", line 153, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.4/urllib/request.py", line 455, in open
    response = self._open(req, data)
  File "/usr/lib/python3.4/urllib/request.py", line 473, in _open
    '_open', req)
  File "/usr/lib/python3.4/urllib/request.py", line 433, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.4/urllib/request.py", line 1202, in http_open
    return self.do_open(http.client.HTTPConnection, req)
  File "/usr/lib/python3.4/urllib/request.py", line 1176, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno -2] Name 
```

**Comment by jackyzy823** [comment]
Name or service not known may be caused by dns query fail?

**Comment by soimort** [comment]
That's on me. works fine with a proxy.

**Comment by soimort** [comment]
Closed via #480
